demos/github/README.md

```diff
@@ -0,0 +1,17 @@
+# Configure gitscm
```

Comment:
"gitscm"? or "github"

Reply:
que?! I don't quite understand your question.
These are two different plugins. I guess `gitscm` is an integrated part of Jenkins.

Reply:
ahh.. gotcha. fixed it.